packages/composite-checkout/package.json

```diff
@@ -9,9 +9,10 @@
 	"sideEffects": false,
 	"scripts": {
 		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && rm -rf dist",
-		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json && copy-assets",
+		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json",
```

Comment:
`copy-assets` wasn't used because this package doesn't have Stylesheets or images. This allows us to drop the dependency to `@automattic/calypso-build` which then allows us to drop `webpack`, `postcss`, `jest` and `enzyme`.